### ISSUE TITLE
User cannot add labels - need to change instructions or github permissions for a user

### ISSUE THREAD

**Issue by debpaul** [SpeciesFileGroup/taxonworks_doc#38]
Overview: In [How to Report A Bug](https://github.com/SpeciesFileGroup/taxonworks_doc/blob/master/CONTRIBUTING.md#how-do-i-report-a-bug-or-feature-request), a user is instructed to "Categorize your issue using a label." This cannot be done with basic permissions in GitHub.

Target Audience: those using TW who want to submit a Bug or Feature Request

Options:
 -[ ] need to change instructions to remove this step, and figure out who will add labels to tickets
 -[ ] else work out for whom we want to up their permissions so they can add labels

**Comment by mjy** [comment]
Good catch. There is a third thing to do here - template out pre-labeled issue type for the major types of issues, that should be a seperate issue somewher linked to this.
